raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -944,8 +957,6 @@ def test_matrix_user_roaming(matrix_transports):
 
     transport0.start(raiden_service0, message_handler0, "")
 
-    assert is_reachable(transport1, raiden_service0.address)
```

Comment:
why this was removed?

Reply:
Because of the change that we made on the user presence, now is reachable it returns always the same because of that so is not a valid call.